tests/apollo/test_skvbc_view_change.py

```diff
@@ -344,7 +352,11 @@ async def _single_vc_with_consecutive_failed_replicas(
             err_msg="Make sure view change has been triggered."
         )
 
-        await tracker.tracked_read_your_writes()
+        with trio.fail_after(seconds=60):
+            while True:
+                with trio.move_on_after(seconds=5):
```

Comment:
Same as above.

Reply:
now I also catch Exception - which includes trio.TooSlowError, so we're covered